### DISCUSSION TITLE
evaluating semantic segmentation using maskrcnn results

### DISCUSSION THREAD

**Discussion by andreaceruti** [facebookresearch/detectron2#4058] [General]
Hi, I would like to evaluate my maskrcnn model also on the semantic segmentation task.
So how could I elaborate the json detections produced by the maskrcnn model (coco_instances_results.json) in order to feed them to the SemSegEvaluator? Anyone could give me some hints?

**Comment by CA4GitHub** [comment]
Did you ever find a solution?

**Reply by andreaceruti** [reply]
@CA4GitHub Yes, look [here](https://github.com/andreaceruti/detectron2_rgbd/blob/main/detectron2/evaluation/coco_evaluation.py#:~:text=%22segm%22%3A-,semantic_segmentation_evaluation(coco_gt%2C%20coco_dt%2C%20output_dir),-%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23%23TODO%3A%20add%20another)
